pkg/cmd/util/output/schedule_printer.go

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright 2017 the Velero contributors.
+Copyright 2017, 2020 the Velero contributors.
```

Comment:
Should this be `2017, 2020` or `2017-2020`?

Reply:
The guidance we've gotten from legal is to add updated years with a comma. If it was updated in 2018 and 2019 too, we'd list them all out.